### PR TITLE
[FLINK-12943][docs]Translate HDFS Connector page into Chinese

### PR DESCRIPTION
**Pull request by aloyszhang**
## What is the purpose of the change

Translate HDFS Connector page into Chinese

## Brief change log

Translate HDFS Connector page into Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? ( no)

### CODE REVIEW

**Comment by aloyszhang** [comment]
@wuchong

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 1]
```diff
@@ -35,16 +34,11 @@ following dependency to your project:
 </dependency>
 {% endhighlight %}
 
-Note that the streaming connectors are currently not part of the binary
-distribution. See
-[here]({{site.baseurl}}/dev/projectsetup/dependencies.html)
-for information about how to package the program with the libraries for
-cluster execution.
+注意连接器目前还不是二进制发行版的一部分，添加依赖、打包配置以及集群运行信息请参考 [here]({{site.baseurl}}/zh/dev/projectsetup/dependencies.html)。
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]: "here" 也要翻译一下

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 2]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
-every minute, and closes any buckets which haven't been written to for over a minute. This
-behaviour can be configured with `setInactiveBucketCheckInterval()` and
-`setInactiveBucketThreshold()` on a `BucketingSink`.
-
-You can also specify a custom bucketer by using `setBucketer()` on a `BucketingSink`. If desired,
-the bucketer can use a property of the element or tuple to determine the bucket directory.
+初始化时只需要一个参数，这个参数表示分桶文件存储的路径。分桶 sink 可以通过指定自定义的 bucketer、 writer 和 batch 值进一步配置。
+
+默认情况下，当数据到来时，分桶 sink 会按照系统时间对数据进行切分，并以 `"yyyy-MM-dd--HH"` 的时间格式给每个桶命名。然后 
+`DateTimeFormatter` 按照 `"yyyy-MM-dd--HH"` 和 JVM 的默认时区格式换当前时间生成分桶的路径。用户可以自定义时区来生成
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]:
```suggestion
`DateTimeFormatter` 按照这个时间格式将当前系统时间以 JVM 默认时区转换成分桶的路径。用户可以自定义时区来生成
```

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 3]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
+分桶的路径。每遇到一个新的日期都会产生一个新的桶。例如，如果时间的格式以分钟为粒度，那么每分钟都会产生一个桶。每个桶都是一个目录，
+目录下包含了几个部分文件：每个 sink 的并发实例都会创建一个部分文件，另外，当这些文件大小太大的时候，sink会产生新的部分文件。
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]:
```suggestion
目录下包含了几个部分文件（part files）：每个 sink 的并发实例都会创建一个属于自己的部分文件，当这些文件太大的时候，sink 会产生新的部分文件。
```

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 4]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
+当一个桶不再活跃时，打开的部分文件会刷盘并且关闭。如果一个桶最近一段时间都没有写入，那么这个桶被认为是不活跃的。sink 默认会每分钟
+检查不活跃的桶、关闭那些超过一分钟没有写入的桶。这些行为可以通过 `BucketingSink` 的 `setInactiveBucketCheckInterval()` 
+和 `setInactiveBucketThreshold()` 进行设置。
+
+可以调用`BucketingSink` 的 `setBucketer()` 方法指定自定义的 bucketer，如果需要的话，也可以使用一个元素或者元组属性来
+决定桶的路径。
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]: 不换行

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 5]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
 
 The default writer is `StringWriter`. This will call `toString()` on the incoming elements
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]: 原文未删除

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 6]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
-and write them to part files, separated by newline. To specify a custom writer use `setWriter()`
-on a `BucketingSink`. If you want to write Hadoop SequenceFiles you can use the provided
-`SequenceFileWriter` which can also be configured to use compression.
+默认的 writer 是 `StringWriter`。数据到达时，通过 `toString()` 方法得到内容，内容以换行符分隔，`StringWriter` 将数据
+内容写入部分文件。可以通过 `BucketingSink` 的 `setWriter()` 指定自定义的 writer。`SequenceFileWriter` 支持写入 Hadoop
+SequenceFiles，并且可以配置是否开启压缩。
 
 There are two configuration options that specify when a part file should be closed
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]: 原文未删除

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 7]
```diff
@@ -65,40 +59,34 @@ input.addSink(new BucketingSink[String]("/base/path"))
...
+关闭部分文件和打开新部分文件的时机可以通过两个配置来确定：
 and a new one started:
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]: 原文未删除

File: docs/dev/connectors/filesystem_sink.zh.md [hunk 8]
```diff
@@ -133,17 +121,15 @@ input.addSink(sink)
...
+上述代码会创建一个 sink，这个 sink 按下面的模式写入桶文件：
 
 {% highlight plain %}
 /base/path/{date-time}/part-{parallel-task}-{count}
 {% endhighlight %}
 
-Where `date-time` is the string that we get from the date/time format, `parallel-task` is the index
-of the parallel sink instance and `count` is the running number of part files that were created
-because of the batch size or batch roll over interval.
+`date-time` 是从日期/时间的字符串，`parallel-task` 是 sink 并发实例的索引，`count` 是因文件大小或者滚动周期而产生的
```
Comment (wuchong) [inline docs/dev/connectors/filesystem_sink.zh.md]:
```suggestion
`date-time` 是我们从日期/时间格式获得的字符串，`parallel-task` 是 sink 并发实例的索引，`count` 是因文件大小或者滚动周期而产生的
```

**Comment by aloyszhang** [comment]
@wuchong 
Thanks for your suggestions and sorry for the mistakes I made. Now this page is fixed.

**Pull request closed without merging**